aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -50,11 +50,22 @@ proposals:
   - name: enable_charge_invariant_violation
     metadata:
       title: "Enable charge_invariant_violation"
```

Comment:
do you mind help me remove this `enable_charge_invariant_violation` section as well now that it has been enabled?

Reply:
Done